community/consistency-check/src/main/java/org/neo4j/consistency/report/ConsistencyReport.java

```diff
@@ -152,6 +152,10 @@ void forCounts( CountsEntry countsEntry,
 
         @Documented( "The schema rule contained in the DynamicRecord chain is of an unrecognized Kind" )
         void unsupportedSchemaRuleKind( SchemaRule.Kind kind );
+
+        @Warning
+        @Documented( "The schema rule contained in the DynamicRecord chain a reference to a schema rule that is not online. It will be rebuilt on next startup." )
```

Comment:
The warning message is a bit wrong here -  at least _is_ word is missing, also it's not that correct since failed indexes will not gonna be rebuilt for example right?